lib/puppet/type/iis_app_pool.rb

```diff
@@ -0,0 +1,12 @@
+require 'puppet/parameter/boolean'
+require_relative '../../puppet_x/puppetlabs/iis/property/string'
+
+Puppet::Type.newtype(:iis_app_pool) do
```

Comment:
Same here. I vote this is changed now to prevent having to change lots of places later